julia_src/GHPGHX.jl

```diff
@@ -0,0 +1,463 @@
+module GHPGHX
+# -------
+export ghp_model
+export get_ghpghx_results_for_reopt
+
+include("ghpghx_inputs.jl")
+
+include("ghpghx_results.jl")
+
+function ghp_model(d)
+    # Load parameters and process from input Dict, to create instance of type InputsStruct
+    p = InputsProcess(d)
+
+    # Run size_borefield for each attempt to SOLVE
+    results = size_borefield(p)
+
+    return results, p
+end
+
+function size_borefield(p)
```

Comment:
Can you please add a docstring to this function and the others giving a high level summary of what each function does?

Reply:
Added for all of the /ghpghx related .jl functions and structs.